### DISCUSSION TITLE
Image property layout='fill' is not filling its parent values

### DISCUSSION THREAD

**Discussion by eddsaura** [vercel/next.js#19246] [Help] [answered]
Sup ppl! 👋🏼

I'm working with Nextjs + Tailwindcss. (Lovely couple 💐 )

I'm trying to put an `Image` component inside a wrapper with fixed width and height and the image fills the most upper parent of the component.

Here we have the code snippet!

```
<nav className='w-full  h-24 bg-pink-200 fixed top-0'>
      <Wrapper className='h-full'>
        <div className='w-56 h-full bg-green-500'>
          <Image src='/img/logo.png' layout='fill' />
        </div>
      </Wrapper>
</nav>
```
And the screenshot. You can see the parent in green and the nav in pink.

<img src="https://i.imgur.com/hZaI0s0.png"/>

It's simple, but I don't know why I can't make it work. I assume the Image component is focused in other use cases, so I will be using normal image, but I still want to learn how to use it, for real haha.

Thanks a lot for your time! 😄

**Comment by Timer** [comment, marked as answer]
Try this!

```tsx
        <div className='relative w-56 h-full bg-green-500'>
          <Image src='/img/logo.png' layout='fill' />
        </div>
```

**Reply by eddsaura** [reply]
Ou dude, I can't until tomorrow morning, but it makes sense, a lot. Thanks!!
